Simplenote/Classes/SPNoteEditorViewController+Extensions.swift

```diff
@@ -705,14 +703,72 @@ extension SPNoteEditorViewController: InterlinkProcessorDelegate {
 }
 
 
+// MARK: - Tags
+//
+extension SPNoteEditorViewController {
+    @objc
+    func configureTagListViewController() {
+        tagListViewController = NoteEditorTagListViewController(note: note)
+        addChild(tagListViewController)
+        let tagView = tagListViewController.view!
```

Comment:
I've noticed `tagListViewController.view` is used in this API + `updateTagListPosition` + `refreshTagsEditor`.

Maybe something like this would be neat?

```
private var tagView: UIView {
    tagListViewController.view!
}
```

Reply:
Updated in https://github.com/Automattic/simplenote-ios/pull/1133/commits/5b14794fc11281504bf5f2161eb176ffcb58c11e
Thanks 👍